dali/kernels/imgproc/convolution/cutlass/threadblock/predicated_tile_iterator.h

```diff
@@ -369,20 +546,320 @@ class PredicatedTileIterator<Shape_, Element_, layout::PitchLinear, AdvanceRank,
   void store(Fragment const &frag) {
     store_with_byte_offset(frag, 0);
   }
+
+ private:
+  /**
+   * @brief Fill the dst with elements of lo[offset...AccessSize-1], hi[0...AccessSize - 1 - offset]
+   */
+  CUTLASS_DEVICE
+  void mux(AccessType &dst, const AccessType &lo, const AccessType &hi, int offset) {
```

Comment:
maybe something misaligned_load or something?

Reply:
It's multiplexing two arrays, as if they were vectors, I would prefer to keep it that way.